### PR TITLE
Fix Tensorboard credentials environment variable

### LINKED ISSUE #1243: `TB_CREDENTIALS` aren't properly detected
Setting `TB_CREDENTIALS` is not enough as credentials is now a required parameter.

cml.dev examples are fundamentally broken due to this.

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Closes #1243

### CODE REVIEW

**Review by dacbd — APPROVED** [review]
lgtm

File: bin/cml.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const { jitsuEventPayload, send } = require('../src/analytics');
 
 const setupOpts = (opts) => {
   const legacyEnvironmentVariables = {
-    TB_CREDENTIALS: 'CML_TENSORBOARD_DEV_CREDENTIALS',
+    TB_CREDENTIALS: 'CML_TENSORBOARD_CREDENTIALS',
```
Comment (DavidGOrtega) [inline bin/cml.js:15]: I have the same error??

```sh
Missing required argument: credentials
```

**Review by DavidGOrtega — CHANGES_REQUESTED** [review]
Seems not to fix it 🙁

File: bin/cml.js (hunk 1 above)
Comment (DavidGOrtega) [reply bin/cml.js:15]: I see that the var is now required

```js
credentials: {
        type: 'string',
        alias: 'c',
        required: true,
        description:
          'TB credentials as json. Usually found at ~/.config/tensorboard/credentials/uploader-creds.json. If not specified will look for the json at the env variable CML_TENSORBOARD_DEV_CREDENTIALS.'
      },
```

**Pull request merged**